packages/lib/src/messaging/Message.ts

```diff
@@ -84,6 +84,8 @@ export async function submitMessage(
         deliveryServiceToken,
         message.metadata.from,
         message.metadata.to,
+        () => {},
+        () => {},
```

Comment:
Do you think we should stop submitting the incoming msg and throw an error here if the pending entry could not be created?

Reply:
I think we should ignore it wouldn't be a problem if the pending entry couldn't be created